### ISSUE TITLE
smartos_prompt_config connectivity checks are kinda useless

### ISSUE THREAD

**Issue by arekinath** [joyent/smartos-live#499]
Currently smartos_prompt_config does some helpful connectivity checks on the DNS and NTP servers you enter. However, if the connectivity checks fail, you do not get a chance to revisit them and correct your dodgy typo, instead you just get UNREACHABLE followed by the installer soldiering on.

Also, the NTP connectivity check timeout seems to be ~5 *minutes*, and waiting for it while staring at your typo is absolutely infuriating.

**Comment by bahamas10** [comment]
agreed. sometimes i will install SmartOS without networking, but know that once it's done and racked it'll have everything it needs.

Simply a lower timeout would be great here
